source/administration/config-settings.rst

```diff
@@ -2345,6 +2358,15 @@ Id Attribute
 | This feature's ``config.json`` setting is ``"IdAttribute": ""`` with string input.                                                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Attribute
+^^^^^^^^^^^^^^^^^
+(Optional) The attribute in the SAML Assertion used to apply a guest role to users in Mattermost.
```

Comment:
Potentially we can link the Guest accounts feature doc here so admins can learn more about the feature?

Reply:
Can definitely do that.

Reply:
Done.